native-code/android/index.md

```diff
@@ -77,6 +77,33 @@ For instructions on how to build and run, see
 [webrtc/examples/androidapp/README][3].
 
 
+### Using Android Studio
+
+  1. Build the project normally:
+
+     ~~~~~ bash
+     ninja -C out/Debug AppRTCMobile
```

Comment:
Do we need to highlight that the out/Debug folder must match the folder used when generating GN files?